packages/server/src/types-builder/base/gql-type.ts

```diff
@@ -1,58 +1,42 @@
-import type { GraphQLNullCheck, GraphQLTypeMap, tGQLBaseTypeAny } from '../../types.ts';
+import type { tGQLBaseTypeAny } from '../../types.ts';
 import type { GraphQLType as GraphQLTypes } from 'graphql';
-import { GraphQLNonNull } from 'graphql';
 
-export abstract class tGQLBase<tGQLType extends tGQLBaseTypeAny, TypescriptType, GraphQLType extends GraphQLTypes> {
+export abstract class tGQLBase<
+	tGQLType extends tGQLBaseTypeAny,
+	TypescriptType,
+	GraphQLType extends GraphQLTypes & { description?: string; deprecationReason?: string }
+> {
 	abstract readonly _class: string;
 	abstract _nullable: boolean;
 	declare _type: TypescriptType;
-	readonly _graphQLType!: GraphQLType;
+	_graphQLType!: GraphQLType;
 	readonly _tGQLType!: tGQLType;
 	readonly name: string | undefined;
 
 	_deprecationReason: string | undefined;
 	_description: string | undefined;
 
-	constructor({ tGQLType, graphQLType, name }: { tGQLType?: tGQLType; graphQLType?: GraphQLTypes; name?: string }) {
+	constructor({ tGQLType, graphQLType, name }: { tGQLType?: tGQLType; graphQLType: GraphQLTypes; name?: string }) {
 		if (tGQLType) this._tGQLType = tGQLType;
 		if (graphQLType) this._graphQLType = graphQLType as GraphQLType;
 		this.name = name;
 	}
 
 	deprecated(reason: string): this {
 		this._deprecationReason = reason;
+		this._graphQLType.deprecationReason = reason;
```

Comment:
It might be this plus line 33 that are causing failures to the field resolver to fail. Field resolvers don't have a graphQLType.